clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -452,9 +453,9 @@ protected void onJoinPrepare(int generation, String memberId) {
 
         // execute the user's callback before rebalance
         ConsumerRebalanceListener listener = subscriptions.rebalanceListener();
-        log.info("Revoking previously assigned partitions {}", subscriptions.assignedPartitions());
+        Set<TopicPartition> revoked = new HashSet<>(subscriptions.assignedPartitions()); //copy
```

Comment:
nits: can we put the comment above the line (as opposed to putting it at the end of the line) for consistency?

Reply:
:+1: